version/version_lib.go

```diff
@@ -0,0 +1,16 @@
+// +build !autogen
+
+// Package version is auto-generated at build-time
+package version
+
+// Default build-time variable for library-import.
+// This file is overridden on build with build-time informations.
+const (
+	GITCOMMIT string = "library-import"
```

Comment:
How did this get merged without following the coding standards?

https://github.com/docker/docker/blob/master/CONTRIBUTING.md#coding-style